app/include/dt-bindings/zmk/bt.h

```diff
@@ -7,9 +7,10 @@
 #define BT_CLR_CMD 0
 #define BT_NXT_CMD 1
 #define BT_PRV_CMD 2
-#define BT_SEL_CMD 3
-#define BT_CLR_ALL_CMD 4
-#define BT_DISC_CMD 5
+#define BT_PRV_DEV_CMD 3
+#define BT_SEL_CMD 4
+#define BT_CLR_ALL_CMD 5
+#define BT_DISC_CMD 6
```

Comment:
Add this at the end with a value of 6 please, otherwise this will break folks using ZMK Studio who've saved these few commands to their keymap.

Also, I am not a huge fan of including `DEV` in the naming here, versus "selection" as a noun instead. I think `BT_PRV_SEL_CMD` is preferable.